system/system_monitor/launch/system_monitor.launch.xml

```diff
@@ -1,15 +1,28 @@
 <launch>
-  <arg name="machine_name" default="$(optenv HOSTNAME localhost)"/>
-  <arg name="config_file" default="$(find system_monitor)/config/system_monitor.yaml"/>
-  <arg name="output" default="log"/>
+  <arg name="config_file" default="$(find-pkg-share system_monitor)/config/system_monitor.yaml"/>
+<!--  <arg name="output" default="log"/>-->
+<!--TODO(yunus.caliskan): Figure out why a variable can't be used for the output type.-->
```

Comment:
@jilaada 
So I would be switching:
```xml
  <node pkg="system_monitor" exec="cpu_monitor" name="cpu_monitor" output="log" respawn="true">
    <param from="$(var config_file)" />
  </node>
```
to:

```xml
  <arg name="output_type" default="log"/>
  <node if="$(eval output_type==log)" pkg="system_monitor" exec="cpu_monitor" name="cpu_monitor" output="screen" respawn="true">
    <param from="$(var config_file)" />
  </node>
  <node if="$(eval output_type==screen)" pkg="system_monitor" exec="cpu_monitor" name="cpu_monitor" output="screen" respawn="true">
    <param from="$(var config_file)" />
  </node>
  <node if="$(eval output_type==both)" pkg="system_monitor" exec="cpu_monitor" name="cpu_monitor" output="both" respawn="true">
    <param from="$(var config_file)" />
  </node>
```

I wasn't able to make it work yet but do you think this is the right direction o is there a more elegant 1-liner i am missing?

Reply:
Hmm that seems like a lot of repetition. After going through the available options in the ROS2 docs, I can't think of an elegant way of doing this either. I think if this feature is not critical then it can be set to `log` for now. I also noticed that in the ros2 launch help output I noticed this: 
```
-a, --show-all-subprocesses-output
                        Show all launched subprocesses' output by overriding their output configuration
                        using the OVERRIDE_LAUNCH_PROCESS_OUTPUT envvar.
```

The user may be able to set the envvar and using this launch argument to pipe the output to `screen`.

Reply:
I don't want to have repetitive codes just for log options so let's use `log` as output since it is set as default value in the original code.

Reply:
In that case I think this can be resolved and merge in, thanks